packages/patrol_cli/lib/src/features/drive/drive_command.dart

```diff
@@ -33,16 +32,20 @@ class DriveCommandConfig with _$DriveCommandConfig {
 class DriveCommand extends StagedCommand<DriveCommandConfig> {
   DriveCommand({
     required DisposeScope parentDisposeScope,
-    required ArtifactsRepository artifactsRepository,
     required DeviceFinder deviceFinder,
     required TestFinder testFinder,
     required TestRunner testRunner,
+    required AndroidDriver androidDriver,
+    required IOSDriver iosDriver,
```

Comment:
Wouldnt it be cleaner if you were to always accept only a single driver (which has some common interface) and make this `TargetPlatform` check when creating this DriveCommand? It would make it easier to mock too, since you would not need to provide always two implementations when in the end only one is used anyways

Reply:
Actually, we need two drivers, because the user can run the tests on more than a single device (e.g 1 device can be an iPhone and the other one can be an Android).